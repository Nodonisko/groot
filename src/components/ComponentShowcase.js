// @flow
import * as React from 'react'
import View from './View'
import Text from './Text'

const ComponentShowcase = () => (
  <View>
    <Text color="red" size="title2" weight="bold" align="center">
      Hello world
    </Text>
  </View>
)

export default ComponentShowcase
