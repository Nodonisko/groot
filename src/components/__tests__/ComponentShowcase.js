// @flow
import 'react-native'
import React from 'react'
import ComponentShowcase from '../ComponentShowcase'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

it('renders correctly with only required props', () => {
  const component = <ComponentShowcase />
  const tree = renderer.create(component).toJSON()
  expect(tree).toMatchSnapshot()
})
