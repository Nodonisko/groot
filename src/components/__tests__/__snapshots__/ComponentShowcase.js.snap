// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with only required props 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "rgb(255, 0, 0)",
          "fontSize": 22,
          "fontWeight": "600",
          "lineHeight": 32,
          "textAlign": "center",
        },
        null,
      ]
    }
  >
    Hello world
  </Text>
</View>
`;
